src/Opdex.Platform.Infrastructure/Clients/CirrusFullNodeApi/Extensions/TransactionErrors.cs

```diff
@@ -50,20 +50,21 @@ private static class General
         internal static bool TryParseFriendlyErrorMessage(string input, out string friendlyErrorMessage)
         {
             friendlyErrorMessage = null;
-            if (IsOverflowException(input))
+            if (!IsOverflowException(input))
             {
-                friendlyErrorMessage = "Value overflow.";
-                return true;
+                return false;
             }
-            return false;
+
+            friendlyErrorMessage = "Value overflow.";
+            return true;
         }
 
         /// <summary>
         /// Attempts to match an overflow exception message.
         /// </summary>
         private static bool IsOverflowException(string input)
         {
-            return input.StartsWith(typeof(OverflowException).FullName);
+            return input.StartsWith(typeof(OverflowException).FullName!);
```

Comment:
What does the `!` at the end do?

Reply:
`Type.FullName` is nullable, the ! indicates to analyzers and the compiler that it is never going to be null.